vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -1071,16 +1071,17 @@ bool AdsImpl::IsSustainingAdInteraction() const {
   return true;
 }
 
-bool AdsImpl::IsStillViewingAd() const {
```

Comment:
It wasn't clear to me that dropping the const on the method was intentional here. Just mentioning it in case it was accidental.

Reply:
good spot 👍 fixed